scipy/signal/tests/test_splines.py

```diff
@@ -149,32 +148,34 @@ def test_symiir1(self, dtype, precision):
         exp_out = exp_out[::-1]
 
         out = symiirorder1(signal, c0, z1, precision)
-        assert_allclose(out, exp_out, atol=4e-6, rtol=6e-7)
+        xp_assert_close(out, exp_out, atol=4e-6, rtol=6e-7)
 
-    @pytest.mark.parametrize('dtyp', [np.float32, np.float64])
+    @pytest.mark.parametrize('dtyp', ['float32', 'float64'])
     def test_symiir1_values(self, dtyp):
         rng = np.random.RandomState(1234)
+        dtyp = getattr(np, dtyp)
         s = rng.uniform(size=16).astype(dtyp)
```

Comment:
can we change `dtyp` to `dtype` in this file? It doesn't *really* matter, but the file uses both spellings which is a bit confusing.

Reply:
OK, will do. 
That said, we should always strive to limit the diff size to the necessary minimum and minimize the churn IMO.